docs/user/security/reporting.asciidoc

```diff
@@ -47,7 +47,7 @@ image::user/security/images/reporting-privileges-example.png["Reporting privileg
 Reporting users typically save searches, create
 visualizations, and build dashboards. They require a space
 that provides read and write privileges in
-*Discover*, *Visualize*, and *Dashboard*.
+*Discover* and *Dashboard*.
```

Comment:
For the Tutorials section, "Tutorial" is ok for the TOC, but we might want to consider making the actual title more specific for SEO reasons.

For the Lens tutorial: 

- How about making it more about what the user is doing:  "Create your first dashboard" or something like that.
- Should it start from Dashboard?

It feels a little overwhelming to me to see all the tutorials on the same page.  What do you think about grouping by type?